src/commands.ts

```diff
@@ -185,6 +187,39 @@ export async function addNewClass(classPath: string, classType: string) {
   console.log("\x1b[36m%s\x1b[0m", "Class added successfully.");
 }
 
+export async function deleteProjectHandler(projectId : string, forced : boolean) {
+  // show prompt if no project id is selected
+  if (typeof projectId === 'string' && projectId.trim().length === 0) {
+    console.log('No project ID specified, select an ID to delete from this list:')
+    const projects = await listProjects();
+    if (projects.length === 0) {
+      return false;
+    }
+
+    const selection = await askQuestion(`Please select project number to delete (0--${projects.length - 1}) [none]: `, "");
```

Comment:
If I just press enter here, it will take the first element from the list. 

```
Please select project number to delete (0--3) [none]: 
Are you sure you want to delete project b810a15c-cf7c-4aa6-a386-7091d4e44129? [Y]/n:

Reply:
Changed the indexing from 0 -> n - 1 to 1 -> n and fixed this.